src/config.rs

```diff
@@ -240,6 +241,12 @@ const DEFAULT_PORTS_DISABLE_PROTOCOL_DETECTION: &[u16] = &[
     3306, // MySQL
 ];
 
+// How long an old TLS config is valid for after falling back on an error.
+//
+// If/when we make the controller CA's validity duration shorter, this
+// should probably track that.
+const DEFAULT_TLS_CONFIG_FALLBACK_TTL: Duration = Duration::from_secs(900); // 15 minutes
```

Comment:
I wasn't sure what an appropriate default value for this was, so I chose 15 minutes arbitrarily. @briansmith, if you have any suggestions, I'd be happy to change this value.